easybuild/easyconfigs/u/UnZip/UnZip-6.0-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'ConfigureMake'
+
+name = 'UnZip'
+version = '6.0'
+
+homepage = 'http://www.info-zip.org/UnZip.html'
+description = """UnZip is an extraction utility for archives compressed
+in .zip format (also called "zipfiles"). Although highly compatible both
+with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's
+own Zip program, our primary objectives have been portability and
+non-MSDOS functionality."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['ftp://ftp.info-zip.org/pub/infozip/src/']
+sources = ['%(namelower)s%(version_major)s%(version_minor)s.tgz']
+patches = [
+    '01-manpages-in-section-1-not-in-section-1l.patch',
+    '03-include-unistd-for-kfreebsd.patch',
+    '04-handle-pkware-verification-bit.patch',
+    '05-fix-uid-gid-handling.patch',
+    '06-initialize-the-symlink-flag.patch',
+    '07-increase-size-of-cfactorstr.patch',
+    '08-allow-greater-hostver-values.patch',
+    '09-cve-2014-8139-crc-overflow.patch',
+    '10-cve-2014-8140-test-compr-eb.patch',
+    '11-cve-2014-8141-getzip64data.patch',
+    '12-cve-2014-9636-test-compr-eb.patch',
+    '13-remove-build-date.patch',
+    '14-cve-2015-7696.patch',
+    '15-cve-2015-7697.patch',
+    '16-fix-integer-underflow-csiz-decrypted.patch',
+    '20-unzip60-alt-iconv-utf8.patch',
```

Comment:
@akesandgren By means of reminder: I would like to see these combined into a single patch that follows the usual naming scheme for patches...